src/software/thunderscope/widget_setup_functions.py

```diff
@@ -130,13 +136,13 @@ def setup_parameter_widget(proto_unix_io, friendly_colour_yellow):
     config = ThunderbotsConfig()
     config.sensor_fusion_config.friendly_color_yellow = friendly_colour_yellow
 
-    def on_change_callback(attr, value, updated_proto):
+    def on_change_callback(attr: Any, value: Any, updated_proto: Message) -> None:
         proto_unix_io.send_proto(ThunderbotsConfig, updated_proto)
```

Comment:
nit: Seems like this should be
```suggestion
    def on_change_callback(attr: Any, value: Any, updated_proto: ThunderbotsConfig) -> None:
        proto_unix_io.send_proto(ThunderbotsConfig, updated_proto)
```